alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+RUN apk --no-cache add curl python
```

Comment:
if `gcloud source` components come installed by default, it means git is a required dependency and we should install it. if it's something user has to `gcloud components install`, then they should probably install it themselves.